vividus/src/main/java/org/vividus/bdd/expression/StringsExpressionProcessor.java

```diff
@@ -67,10 +71,32 @@ public StringsExpressionProcessor(ILocationProvider locationProvider)
                     .encodeToString(ResourceUtils.loadResourceAsByteArray(input))),
             new UnaryExpressionProcessor("decodeFromBase64",  input -> new String(Base64.getDecoder()
                     .decode(input.getBytes(UTF_8)), UTF_8)),
-            new UnaryExpressionProcessor("encodeToBase64",    input -> new String(Base64.getEncoder()
-                    .encode(input.getBytes(UTF_8)), UTF_8)),
-            new UnaryExpressionProcessor("anyOf",             StringsExpressionProcessor::anyOf)
-            ));
+            new UnaryExpressionProcessor("encodeToBase64",    input -> encodeToBase64(input.getBytes(UTF_8))),
+            new UnaryExpressionProcessor("anyOf",             StringsExpressionProcessor::anyOf),
+            new UnaryExpressionProcessor("toBase64Gzip",      StringsExpressionProcessor::toBase64Gzip)
+        ));
+    }
+
+    private static String encodeToBase64(byte[] input)
+    {
+        byte[] encodedInput = Base64.getEncoder().encode(input);
+        return new String(encodedInput, UTF_8);
+    }
+
+    private static String toBase64Gzip(String input)
+    {
+        try (ByteArrayOutputStream baos = new ByteArrayOutputStream())
```

Comment:
multi try?

Reply:
not possible, it's needed to close `GZIPOutputStream` before getting byte array from `ByteArrayOutputStream `